joblib/_parallel_backends.py

```diff
@@ -147,18 +149,21 @@ def retrieval_context(self):
         """
         yield
 
-    @classmethod
-    def limit_clib_threads(cls, n_threads=1):
-        """Initializer to limit the number of threads used by some C-libraries.
+    def get_max_num_threads_vars(self, n_threads=1):
```

Comment:
Maybe this should be kept private. I don't see the point in making it public since it's never called by `Parallel` directly.